mino/minogrpc/traffic.go

```diff
@@ -6,32 +6,44 @@ import (
 	"io"
 	"os"
 	"regexp"
+	"sync"
 
 	"github.com/golang/protobuf/proto"
+	"go.dedis.ch/fabric/encoding"
 	"go.dedis.ch/fabric/mino"
 )
 
 var eachLine = regexp.MustCompile(`(?m)^(.+)$`)
 
+var counter = atomicCounter{}
+
 // traffic is used to keep track of packets traffic in a server
 type traffic struct {
```

Comment:
Can you either test this file, or move it to an internal/testing package ?

Reply:
Yes testing is planed. Added in the TODO because for now this is still very prone to change.